spec/collections_publisher/archiving_child_topic_spec.rb

```diff
@@ -1,17 +1,17 @@
-feature "Archiving a child topic on Collections Publisher", collections: true, collections_publisher: true do
+feature "Archiving a Specialist topic page's subtopic on Collections Publisher", collections: true, collections_publisher: true do
```

Comment:
nitpick
Could it just say `Archiving a Specialist subtopic on Collections Publisher`